diagnostics/README.md

```diff
@@ -19,7 +19,18 @@ NGTF uses the std error to output its logs, so it is necessary to pipe it correc
 
 ### A full dump
 To get a **full** dump use the following set of flags
-```NGRAPH_ENABLE_SERIALIZE=1 NGRAPH_CPU_TRACING=1 NGRAPH_TF_VLOG_LEVEL=5 NGRAPH_GENERATE_GRAPHS_PBTXT=1 NGRAPH_TF_LOG_PLACEMENT=1 NGRAPH_TF_VALIDATE_CLUSTER_GRAPHS=1 NGRAPH_TF_DUMP_GRAPHS=1 python run_TF_network.py > log.txt 2>&1```
+```NGRAPH_ENABLE_SERIALIZE=1 NGRAPH_CPU_TRACING=1 NGRAPH_TF_VLOG_LEVEL=5 NGRAPH_TF_LOG_PLACEMENT=1 NGRAPH_TF_DUMP_CLUSTERS=1 NGRAPH_TF_DUMP_GRAPHS=1 python run_TF_network.py > log.txt 2>&1```
+
+## Debug flags
+* ```NGRAPH_ENABLE_SERIALIZE=1```: Generate nGraph level serialized graphs .json
+* ```NGRAPH_CPU_TRACING=1```: Generate nGraph level function timelines
+* ```NGRAPH_TF_VLOG_LEVEL=5```: Generate ngraph-tf logging info for different passes
+* ```NGRAPH_GENERATE_GRAPHS_PBTXT=1```: Generate .pbtxt files for different phases in ngraph-tf bridge
+* ```NGRAPH_TF_LOG_PLACEMENT=1```: will generate op placement log to stdout
+* ```NGRAPH_TF_DUMP_CLUSTERS=1```: Dumps Encapsulated TF Graphs: ngraph_cluster_<cluster_num>
+* ```NGRAPH_TF_DUMP_GRAPHS=1```: dumps TF graphs for different passes : precapture, capture, unmarked, marked, clustered, declustered, encapsulated
+* ```TF_CPP_MIN_VLOG_LEVEL=1```: Enables TF CPP Logs 
+* ```NGRAPH_TF_DUMP_DECLUSTERED_GRAPHS=1```: To view TF computation graph with colored nodes indicating clusters
```

Comment:
Moved the explanation of debug Flags close to where there are first used. 
NGRAPH_TF_VALIDATE_CLUSTER_GRAPHS was the old flag for NGRAPH_TF_DUMP_CLUSTERS. We missed updating it here. Removed NGRAPH_GENERATE_GRAPHS_PBTXT=1 from full log as NGRAPH_TF_DUMP_GRAPHS=1 would created both .pbtxt and .dot